addons/mass_mailing/models/mail_mail.py

```diff
@@ -68,7 +68,7 @@ def _get_unsubscribe_url(self, cr, uid, mail, email_to, msg=None, context=None):
                 'params': urllib.urlencode({'db': cr.dbname, 'res_id': mail.res_id, 'email': email_to})
             }
         )
-        return '<small><a href="%s">%s</a></small>' % (url, msg or 'Click to unsubscribe')
+        return url, msg
```

Comment:
Previously we were manually appending the the un-subscription link after mail body with help of this method. But now, as asked in specification, we will already have html content for the link in mail body and we will just set the link there, so we won't need to return html content now.